app/ui-utils/client/lib/openRoom.js

```diff
@@ -91,9 +91,11 @@ export const openRoom = async function (type, name, render = true) {
 		} catch (error) {
 			c.stop();
 			if (type === 'd') {
-				const result = await callWithErrorHandling('createDirectMessage', ...name.split(', '));
-				if (result) {
+				try {
+					const result = await callWithErrorHandling('createDirectMessage', ...name.split(', '));
```

Comment:
`callWithErrorHandling` is redundant here as you still have to catch the exception. Better to use `call`:

```suggestion
					const { rid } = await call('createDirectMessage', ...name.split(', '));
```